### PR TITLE
[AC-9988] Opsworks web python ERROR: bash[installing python3.6]

### PR DESCRIPTION
**Pull request by AmosWels**
#### What does this PR do?
- Resolves  the Opsworks web python ERROR: bash[installing python3.6]

#### How should the reviewer navigate to the affected pages? (if appropriate)
- Visit https://us-east-1.console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/75bae516-3bb7-4086-967b-93010ab619bc/instances and see that the instance is successfully set up

#### Any extra settings needed to be configured in the admin interface?
- N/A

#### Any background context you want to provide?
- N/a

#### What are the relevant Jira stories?
- https://masschallenge.atlassian.net/browse/AC-9988

#### Screenshots (if appropriate)
- N/a

[Sibling PR](https://github.com/masschallenge/python/pull/6)

### CODE REVIEW

**Review by iankigen — COMMENTED** [review]
Unable to setup a new instance
https://us-east-1.console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/885c6677-240a-425d-8244-6da3c48cd404/instances/fab19c4c-7280-499f-a382-813f9ddd53f5/log/9a3d0265-8811-419b-94e6-50bfc5d27c24
```
[2023-01-10T11:34:18+00:00] ERROR: bash[install python3.6] (python::python line 20) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '100'
---- Begin output of "bash" -x "/tmp/chef-script20230110-5538-17u466q" ----
STDOUT: Reading package lists...
Building dependency tree...
Reading state information...
The following extra packages will be installed:
libpython3.6 libpython3.6-dev libpython3.6-minimal libpython3.6-stdlib
python3.6-minimal
Suggested packages:
python3.6-venv python3.6-doc binfmt-support
The following NEW packages will be installed:
libpython3.6 libpython3.6-dev libpython3.6-minimal libpython3.6-stdlib
python3.6 python3.6-dev python3.6-minimal
0 upgraded, 7 newly installed, 0 to remove and 74 not upgraded.
Need to get 8687 kB of archives.
After this operation, 39.9 MB of additional disk space will be used.
WARNING: The following packages cannot be authenticated!
libpython3.6-minimal libpython3.6-stdlib libpython3.6 python3.6-minimal
libpython3.6-dev python3.6 python3.6-dev
STDERR: + sudo apt install python3.6 python3.6-dev -y
 
WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.
 
E: There are problems and -y was used without --force-yes
---- End output of "bash" -x "/tmp/chef-script20230110-5538-17u466q" ----
Ran "bash" -x "/tmp/chef-script20230110-5538-17u466q" returned 100

**Comment by AmosWels** [comment]
> Unable to setup a new instance https://us-east-1.console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/885c6677-240a-425d-8244-6da3c48cd404/instances/fab19c4c-7280-499f-a382-813f9ddd53f5/log/9a3d0265-8811-419b-94e6-50bfc5d27c24
> 
> ```
> [2023-01-10T11:34:18+00:00] ERROR: bash[install python3.6] (python::python line 20) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '100'
> ---- Begin output of "bash" -x "/tmp/chef-script20230110-5538-17u466q" ----
> STDOUT: Reading package lists...
> Building dependency tree...
> Reading state information...
> The following extra packages will be installed:
> libpython3.6 libpython3.6-dev libpython3.6-minimal libpython3.6-stdlib
> python3.6-minimal
> Suggested packages:
> python3.6-venv python3.6-doc binfmt-support
> The following NEW packages will be installed:
> libpython3.6 libpython3.6-dev libpython3.6-minimal libpython3.6-stdlib
> python3.6 python3.6-dev python3.6-minimal
> 0 upgraded, 7 newly installed, 0 to remove and 74 not upgraded.
> Need to get 8687 kB of archives.
> After this operation, 39.9 MB of additional disk space will be used.
> WARNING: The following packages cannot be authenticated!
> libpython3.6-minimal libpython3.6-stdlib libpython3.6 python3.6-minimal
> libpython3.6-dev python3.6 python3.6-dev
> STDERR: + sudo apt install python3.6 python3.6-dev -y
>  
> WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.
>  
> E: There are problems and -y was used without --force-yes
> ---- End output of "bash" -x "/tmp/chef-script20230110-5538-17u466q" ----
> Ran "bash" -x "/tmp/chef-script20230110-5538-17u466q" returned 100
> ```

Some more additions have been made to ensure that the error is resolved. There were dependencies in the Python repo [here](https://github.com/masschallenge/python/commit/756d31eb1cb6f9a75d97d46c004660e8d6aa3148) that were commented out 

And in the Berksfile - We specifically pointed to Branch AC-9988 for the python installation. You may view it [here](https://github.com/masschallenge/opsworks-web-python/blob/AC-9988/Berksfile#L26)

**Comment by iankigen** [comment]
Still failing: https://us-east-1.console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/885c6677-240a-425d-8244-6da3c48cd404/instances/a8e82564-4cf9-438d-a8ab-40791f50c5f8/log/b9a191a0-eb2f-4348-ae8e-b015ed77c822

**Comment by AmosWels** [comment]
> Still failing: https://us-east-1.console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/885c6677-240a-425d-8244-6da3c48cd404/instances/a8e82564-4cf9-438d-a8ab-40791f50c5f8/log/b9a191a0-eb2f-4348-ae8e-b015ed77c822

There is a sibling PR that addressed the failing build. [Link](https://github.com/masschallenge/python/pull/6)

File: Berksfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ cookbook 'ulimit', '0.3.2'
 cookbook 'nfs', '1.0.0'
 cookbook 'line', '~> 0.5.1'
 cookbook 'postfix', '3.1.8'
-cookbook 'python', git: 'https://github.com/masschallenge/python'
+cookbook 'python', git: 'https://github.com/masschallenge/python',  branch: 'AC-9988'
```
Comment (iankigen) [inline Berksfile]: remove branch once Python PR is merged

**Review by iankigen — APPROVED** [review]

**Review by kenware — APPROVED** [review]

**Pull request merged**